index.bs

```diff
@@ -687,10 +692,10 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 This section normatively specifies the API for creating and using [=public key credentials=]. The basic
 idea is that the credentials belong to the user and are managed by an authenticator, with which the [=[WRP]=] interacts through the
-client (consisting of the browser and underlying OS platform). Scripts can (with the [=user consent|user's consent=]) request the
+[=client platform=]. Scripts can (with the [=user consent|user's consent=]) request the
```

Comment:
while we're here: 
Scritps -> [=[RP]=] scripts

Reply:
Done.